elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -46,7 +46,7 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
   border-color: transparent;
   border-bottom-width: pfe-local(BorderWidth);
   // Inner style
-  background-color: pfe-local(BackgroundColor);
+  background-color: pfe-local(BackgroundColor--inactive, $fallback: none); //@todo update to use opacity
```

Comment:
Instead of hardcoding the fallback here, I suggest you add it to the definition map at the top.  The TODO syntax should be: `// TODO` to pick up by the tool

Reply:
if I move it to the map, we lose the inactive background color for earth tabs:
![image](https://user-images.githubusercontent.com/456863/96476629-790c8e80-1203-11eb-9460-6ba5f5f0c4c9.png)

Reply:
I would suggest earth has the fallback override.  In my opinion, the map should contain the default values and states to make it easier to understand.  I'm fine if you want to include variants in the top map but they should be called out if they are.